official-antistasi-community/A3-Antistasi
# Update all Current Templates

## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Add more Magazines to Weapon Arrays,
Add GL Ammo to Weapon Arrays,
Add Multiple Rocket types to Launcher Arrays,
Update Breaching charges,
Ensure MineAT and MineApers is using the correct type,
Update Police and Squadleader Loadouts,
Swapped out Artillery/Mortar on LDF and AAF to avoid possible Side Issue with Support rework,
Add UnitTraits to Templates, only Engineer, Medic and Explosive Expert right now,
Deleted TKA_Arid as its unused now.

    

### Please specify which Issue this PR Resolves.
closes none

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Load Templates and check on the Gear of each Unit aswell as a bit of Playtesting in general.

********************************************************
Notes:
